Sources/SwiftCourses/HomeTask1.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  File.swift
+//
+//
+//  Created by George on 03.11.22.
+//
+
+import Foundation
+
+public class HomeTask1: HometaskBase {
+  public override func tasks() -> TaskList {
+    return [task1, task2, task3]
+  }
+
+  func task1() -> TaskResult {
+    let firstString: String = "I can"
+    let secondString: String = "code"
+
+    return ["\(firstString) \(secondString)!"]
```

Comment:
Why you need an array here?

Reply:
This is not related to this particular task, I found it convenient to organize the project so that each class implements a separate homework, as a result returns an array of strings (output).
The main idea is to cover the tests and plus a neater formatting of the print.
So in unit tests I could just call this function and compare output with expected strings...

Output example:
<img width="941" alt="Screenshot 2022-11-08 at 22 56 11" src="https://user-images.githubusercontent.com/8019484/200673639-8e8ff4be-8f30-490e-b101-60ca5078b2d6.png">